src/main/java/seedu/guilttrip/ui/ResultDisplay.java

```diff
@@ -21,6 +24,8 @@ public ResultDisplay() {
     }
 
     public void setFeedbackToUser(String feedbackToUser) {
+        logger.info("Sending feedback: " + feedbackToUser);
+        logger.info("filling inner parts");
```

Comment:
Why is this logger logging `filling inner parts` here?

Reply:
can delete